src/api/IBMi.ts

```diff
@@ -16,6 +16,8 @@ export interface MemberParts extends IBMiMember {
   basename: string
 }
 
+const SERVER_SETTINGS_PATH = `/.vscode/connections.json`;
```

Comment:
This is my only concern for this PR - the location of the server setting JSON file!

I don't like creating a hidden directory in the root - especially since the recommendation from IBM is to remove write capability for users for root. I think it's better to store the directory in `/etc` where server configuration files are normally placed.

For the name of the directory and JSON file, I suggest the following:

```suggestion
const SERVER_SETTINGS_PATH = `/etc/Code4IBMi/common_server_settings.json`;
```

This is more intuitive IMHO - the file contains server settings common for all users of the Code for IBM i extension on this server.

WDYT? Too long filename?

Reply:
I will shorted to simply `settings.json`!